source/material_model/rheology/elasticity.cc

```diff
@@ -485,8 +794,14 @@ namespace aspect
       calculate_viscoelastic_viscosity (const double viscosity,
                                         const double shear_modulus) const
       {
+        // The elastic viscosity has been scaled with the timestep ratio $\frac{\Delta t_c}{\Delta t_{el}}$.
+        // The viscous viscosity has not been scaled yet, so we do it here.
+        // Scaling both viscosities with the timestep ratio before computing the effective
+        // viscoelastic viscosity equals scaling the viscoelastic viscosity computed from
+        // the unscaled elastic and viscous viscosity.
```

Comment:
Perhaps this is a good place to explain how this works. From our paper:
<img width="598" alt="image" src="https://github.com/geodynamics/aspect/assets/8144905/f8988d96-8c11-4f42-b7b8-a11f62011ecf">